configure.ac

```diff
@@ -2273,6 +2273,14 @@ then
         AC_MSG_ERROR([cannot enable eccencrypt without enabling hkdf.])
     fi
     AM_CFLAGS="$AM_CFLAGS -DHAVE_ECC_ENCRYPT"
+    if test "$ENABLED_ECC_ENCRYPT" = "old"
```

Comment:
Please show these new options in the help for `--enable-eccencrypt`

Reply:
Done